File: src/main/java/ru/testproject/catalog/to/ProductTO.java
package ru.testproject.catalog.to;

import io.swagger.annotations.ApiModel;

import javax.validation.constraints.NotBlank;
import java.io.Serializable;

@ApiModel
public class ProductTO implements Serializable {
    @NotBlank(message = "Name is mandatory")
    public String name;
    @NotBlank(message = "Category name is mandatory")
    public String category;
}

File: src/main/kotlin/ru/petproject/catalog/to/ProductTO.kt
package ru.petproject.catalog.to

import jakarta.validation.constraints.NotBlank
import java.io.Serializable

class ProductTO : Serializable {
    @field:NotBlank(message = "Name is mandatory")
    var name: String? = null

    @field:NotBlank(message = "Category name is mandatory")
    var category: String? = null
}
